### PR TITLE
Connects #38:  Add pagination to index pages.

### PR DESCRIPTION
**Pull request by ccostino**
This changeset adds support for pagination on all of the index pages.  It also truncates the message body on the sent scheduled messages listing to help with the display of the page.

### CODE REVIEW

File: config/initializers/kaminari_config.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Kaminari.configure do |config|
+  config.default_per_page = 10
```
Comment (jessieay) [inline config/initializers/kaminari_config.rb]: can we add a quick test where we stub this number to be something like 1 and make sure that a second record doesn't show? Not a huge deal, just would be nice to have a smoke test that pagination is working

Comment (ccostino) [inline config/initializers/kaminari_config.rb]: Can do!

File: spec/features/view_scheduled_messages_spec.rb [hunk 2]
```diff
@@ -18,6 +18,38 @@
     expect(page).to have_content(second_scheduled_message.time_of_day.strftime("%l:%M %p"))
   end
 
+  scenario "sees pagination controls" do
+    allow(Kaminari.config).to receive(:default_per_page).and_return(1)
```
Comment (jessieay) [inline spec/features/view_scheduled_messages_spec.rb]: awesome

**Comment by jessieay** [comment]
SHIP IT!

**Pull request merged**